modules/social_features/social_group/modules/social_group_invite/social_group_invite.routing.yml

```diff
@@ -0,0 +1,11 @@
+social_group_invite.invitation.accepted:
+  path: '/social-group-invite/{group_content}/accepted'
```

Comment:
I think it would be better to rename URL and method from `accepted` to just `accept`

Reply:
done